doc/source/tune.rst

```diff
@@ -27,7 +27,7 @@ Features
 
 *  Mix and match different hyperparameter optimization approaches - such as using `HyperOpt with HyperBand`_ or `Nevergrad with HyperBand`_.
 
-*  Visualize results with `TensorBoard <https://www.tensorflow.org/get_started/summaries_and_tensorboard>`__, `parallel coordinates (Plot.ly) <https://plot.ly/python/parallel-coordinates-plot/>`__, and `rllab's VisKit <https://media.readthedocs.org/pdf/rllab/latest/rllab.pdf>`__.
+*  Visualize results with `TensorBoard <https://www.tensorflow.org/get_started/summaries_and_tensorboard>`__ and `rllab's VisKit <https://media.readthedocs.org/pdf/rllab/latest/rllab.pdf>`__.
```

Comment:
Btw, I would maybe change the viskit link to https://github.com/vitchyr/viskit, since I don't think rllab's original viskit is maintained anymore. @vitchyr's viskit is a for of the original one and has some bug fixes and minor improvements.